storage-provider-guide/tutorials.md

```diff
@@ -147,6 +161,14 @@ docker run --rm \
           --entrypoint /es-node/init.sh \
           ghcr.io/ethstorage/es-node:v0.1.16 \
           --l1.rpc <el_rpc>
+
+# SWC
+docker run --rm \
+          -v ./es-data:/es-node/es-data \
+          -v ./zkey:/es-node/build/bin/snark_lib/zkey \
+          -e ES_NODE_STORAGE_MINER=<miner> \
+          --entrypoint /es-node/init-l2.sh \
+          ghcr.io/ethstorage/es-node:v0.1.16
```

Comment:
v0.1.16 is not support l2 beta testnet, do we want to release a new version?

Reply:
Maybe after https://github.com/ethstorage/es-node/pull/360 and https://github.com/ethstorage/es-node/pull/355 merged?

Reply:
Due to potential backward incompatibility issues on the current testnets, especially the L1 public testnet, we may need to defer merging https://github.com/ethstorage/es-node/pull/355 into the upcoming release.

We will release v0.1.17 and change the version references in the doc in another PR.